src/SFA.DAS.AssessorService.Application.Api/Orchestrators/IOrganisationSearchOrchestrator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SFA.DAS.AssessorService.Api.Types.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.Api.Orchestrators
+{
+    public interface IOrganisationSearchOrchestrator
+    {
+        bool IsValidEpaOrganisationId(string organisationIdToCheck);
+        bool IsValidUkprn(string stringToCheck, out int ukprn);
+        Task<IEnumerable<OrganisationSearchResult>> OrganisationSearchByEpao(string epaoId);
+        Task<IEnumerable<OrganisationSearchResult>> OrganisationSearchByNameOrCharityNumberOrCompanyNumber(string searchTerm);
+        Task<IEnumerable<OrganisationSearchResult>> OrganisationSearchByUkprn(int ukprn);
```

Comment:
I think these are slightly better names:

```suggestion
        Task<IEnumerable<OrganisationSearchResult>> FindOrganisationByEpao(string epaoId);
        Task<IEnumerable<OrganisationSearchResult>> FindOrganisationByNameOrCharityNumberOrCompanyNumber(string searchTerm);
        Task<IEnumerable<OrganisationSearchResult>> FindOrganisationByUkprn(int ukprn);
```

Reply:
The search matches the search in the orchestration and controller

Reply:
How about this then?
```suggestion
        Task<IEnumerable<OrganisationSearchResult>> SearchOrganisationByEpao(string epaoId);
        Task<IEnumerable<OrganisationSearchResult>> SearchOrganisationByNameOrCharityNumberOrCompanyNumber(string searchTerm);
        Task<IEnumerable<OrganisationSearchResult>> SearchOrganisationByUkprn(int ukprn);
```